### PR TITLE
Fixes#2

### PR DESCRIPTION
**Pull request by sharathnair9999**
Made few CSS fixes with respect to responsive layout. 
Made a Responsive Navbar
Changing text color according to the background color of the Note Container

### CODE REVIEW

**Review by ashishnnnnn — APPROVED** [review]
Code seems Understandable..
Can be Merged :+1:

File: src/components/RichTextEditor/RichTextEditor.css [hunk 1]
```diff
@@ -1,97 +1,122 @@
...
+
+.text-editor p {
+  min-height: 20rem !important;
+}
+.ql-container.ql-snow{
+  border: none !important;
+}
+.ql-toolbar.ql-snow{
+  border: none !important;
+}
```
Comment (AmishMishra11) [inline src/components/RichTextEditor/RichTextEditor.css:20]: its not a good thing to have important, maybe find another way on implementing these properties

**Review by AmishMishra11 — APPROVED** [review]
Good use of CSS variables

File: src/components/RichTextEditor/RichTextEditor.css (hunk 1 above)
Comment (sharathnair9999) [reply src/components/RichTextEditor/RichTextEditor.css:20]: ok will try that out

**Pull request merged**